packages/main/src/themes/Toolbar.css

```diff
@@ -6,23 +6,10 @@
 	justify-content: flex-end;
 	padding: 0 var(--_ui5-toolbar-padding-left) 0 var(--_ui5-toolbar-padding-right);
 	box-sizing: border-box;
-	background: var(--ui5-toolbar-Background);
 	color: var(--sapButton_TextColor);
```

Comment:
Also do we need this color here?

Reply:
Correct, the color: `var(--sapButton_TextColor);` is also an extra.